core/src/main/java/org/mule/DefaultMuleMessage.java

```diff
@@ -475,18 +468,6 @@ public void setInboundProperty(String key, Object value, DataType<?> dataType)
     }
 
     @Override
-    public void setInvocationProperty(String key, Object value)
```

Comment:
They are still used widely in tests to set inbound properties. Best to clean this up in a different issue/pr/